package.json

```diff
@@ -22,6 +22,7 @@
     "mongoose": "^6.9.0",
     "next": "^13.1.6",
     "next-auth": "^4.18.10",
+    "next-themes": "^0.2.1",
```

Comment:
Is this needed, as we use Tailwind and that has a dark mode theme built in already https://tailwindcss.com/docs/dark-mode

Reply:
I have used nex-themes to persist the theme that the user has selected.

Initially it will be the user's system theme then it stores the theme in local storage.

This seemed to be pretty standard in the examples I saw.

Reply:
If we can avoid including extra libraries when the libraries we already have can do this, it would be better much better for the project

Reply:
I'll check if we can do this natively. Will update

Reply:
Great, thank you 👍

Reply:
Updated with localstorage hook